lib/semantic_puppet/dependency.rb

```diff
@@ -64,10 +73,12 @@ def query(modules)
     # @param graph [Graph] the root of a dependency graph
     # @return [Array<ModuleRelease>] the list of releases to act on
     def resolve(graph)
+      @module_dependencies, @satisfieds = nil
```

Comment:
Should this be `@module_dependencies = @satisfieds = nil`?

Reply:
I think it's the more idiomatic way, but it escaped my mind when writing the code.

Did some searching and apparently this way of assignment works for `nil` due to the default value of a variable being `nil` if not assigned explicitly: https://stackoverflow.com/a/34813102, so it shouldn't be an issue.